Challenge Solved

gcoayla opened the pull request:
![ss](https://user-images.githubusercontent.com/13878014/126021637-e7ee1416-f2bc-411c-bd1e-95cc504085a9.png)


Live Preview: https://gallant-clarke-b6aa39.netlify.app/

gcoayla committed:
Challenge Solved

gcoayla committed:
Screenshot

gcoayla committed:
New design Solved

gcoayla committed:
Readme Update

gcoayla committed:
Search background fix

laurieesc commented:
**Areas of improvement:**
1. Alignment, spacing, padding, and margins are not consistent with the original design
2. Icons used are different than the original design
3. Typography has inconsistent styling

Nice touch with the cursor changing on top of interactive elements

gcoayla committed:
Suggestions applied

Fixed font styling
Fixed alingment, spacing and margin
Used icons from the desing

gcoayla committed:
Update ss.png
